lcc_members/views/profile_portal_templates.xml

```diff
@@ -70,21 +70,183 @@
             <t t-call="portal.portal_record_layout">
                 <t t-set="card_header">
                     <h5 class="mb-0">
-                        <small class="text-muted">Profile - </small><span t-field="profile.name"/>
+                        <strong><span t-field="profile.name"/></strong>
+                        <small class="text-muted"> - <span t-field="profile.partner_profile"/></small>
                     </h5>
+                    <div t-if="profile.partner_profile and (profile.partner_profile.name == 'Position')"> <!-- TODO: change partner_profile.name by partner_profile.ref when code available -->
+                        <h5 class="mb-0">
+                            <span t-field="profile.parent_id"/>
+                        </h5>
+                        <br/>
+                        <ul class="col-12 col-md-6 pb-2" style= "list-style-type:none">
+                            <li><span t-field="profile.street"/></li>
+                            <li><span t-field="profile.zip"/></li>
+                            <li><span t-field="profile.city"/></li>
+                            <li><span t-field="profile.country_id"/></li>
```

Comment:
It is because that field on line #85 is not to be editable. It is for the position profile, and the address comes from the parent_id (the society). In the public profile part of the code (line #185), I use the code you wrote above :-)